en-am/jw300-amharic-baseline/English_to_Amharic.ipynb

```diff
@@ -0,0 +1,1166 @@
+{
```

Comment:
<p>Can we add a drop_duplicates like we have in the base message? </p><p><code></code><code></p><p>df = df.drop_duplicates()</p><p></code><code></code></p><p>before you do the splits? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/jaderabbit/masakhane/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>